app/views/cycle_counts/_cycle_count.html.erb

```diff
@@ -1,6 +1,11 @@
 <tr>
   <td><%= cycle_count.location.description %></td>
   <td><%= cycle_count.formatted_requested_date %></td>
+  <%unless cycle_count.pallets.count > 0 %>
+    <td><%= link_to 'Add Pallets', cycle_count_pallets_path(cycle_count) %></td>
+  <% else %>
+    <td><%= link_to 'Show Pallets', cycle_count_pallets_path(cycle_count) %></td>
+  <% end %>
   <td><%= link_to 'Show', cycle_count_path(cycle_count) %></td>
```

Comment:
Same thing.  Just a way to tell if there are pallets associated with the cycle count or not.